introduction.md

```diff
@@ -0,0 +1 @@
+My name is Bryan and I work as a UX / UI Developer at Allegiant Travel Company in Las Vegas, NV. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.